### PR TITLE
Add comment to make what setDuration for effect is in more obvious

### PR DESCRIPTION
**Pull request by Zedstar16**
## Introduction

Noob might not realise that setDuration() for an effect instance is in ticks rather than seconds,
Comment makes it more obvious

### CODE REVIEW

**Review by dktapps — COMMENTED** [review]
looks OK, but pls use tabs for indentation instead of spaces

File: src/pocketmine/entity/EffectInstance.php
Comment (SOF3) [inline src/pocketmine/entity/EffectInstance.php]: and the second line too...

**Review by SOF3 — CHANGES_REQUESTED** [review]

File: src/pocketmine/entity/EffectInstance.php
Comment (dktapps) [reply src/pocketmine/entity/EffectInstance.php]: why is this marked as resolved?

**Comment by dktapps** [comment]
Closing this due to too much hassle for such a small change; it can be applied by a maintainer without waiting for review comments to be addressed.

**Pull request closed without merging**
